Jenkinsfile

```diff
@@ -0,0 +1,9 @@
+elifeLibrary {
+    stage 'Checkout', {
+        checkout scm
+    }
+
+    elifeVariants(['lowest', 'default'], { dependencies ->
```

Comment:
Not a library, so not needed.

Reply:
probably will implement the `develop`|`master` merge